src/mono/mono.proj

```diff
@@ -642,6 +643,43 @@
       <_MonoCFLAGS Include="-fexceptions" />
       <_MonoCXXFLAGS Include="-fexceptions -s DISABLE_EXCEPTION_CATCHING=0" />
       <_MonoAC_VARS Include="ac_cv_func_shm_open_working_with_mmap=no" />
+
+      <!-- AOT cross-compiler configuration -->
+      <_MonoAotCrossConfigureParams Include="--with-core=only" />
+      <_MonoAotCrossConfigureParams Include="--enable-maintainer-mode" />
+      <_MonoAotCrossConfigureParams Include="--enable-compile-warnings" />
+      <_MonoAotCrossConfigureParams Include="--prefix=$(MonoObjDir)cross/out" />
+      <_MonoAotCrossConfigureParams Include="--disable-boehm" />
+      <_MonoAotCrossConfigureParams Include="--disable-btls" />
+      <_MonoAotCrossConfigureParams Include="--disable-iconv" />
+      <_MonoAotCrossConfigureParams Include="--disable-libraries" />
+      <_MonoAotCrossConfigureParams Include="--disable-mcs-build" />
+      <_MonoAotCrossConfigureParams Include="--disable-nls" />
+      <_MonoAotCrossConfigureParams Include="--enable-dtrace=no" />
+      <_MonoAotCrossConfigureParams Include="--enable-icall-symbol-map" />
+      <_MonoAotCrossConfigureParams Include="--enable-minimal=com,remoting" />
+      <_MonoAotCrossConfigureParams Include="--disable-crash-reporting" />
+      <_MonoAotCrossConfigureParams Include="--with-cooperative-gc=no" />
+      <_MonoAotCrossConfigureParams Include="--enable-hybrid-suspend=no" />
+      <_MonoAotCrossConfigureParams Include="--target=wasm32-unknown-none" />
+      <_MonoAotCrossConfigureParams Include="--with-cross-offsets=$(MonoObjDir)cross/offsets-wasm32-unknown-none.h" />
+	  <!-- FIXME: LLVM -->
+
+      <_MonoAotCrossCFLAGS Include="-O2" />
+      <_MonoAotCrossCFLAGS Include="-g" />
+      <_MonoAotCrossCFLAGS Include="-m64" />
+
+      <HostDylibSuffix Include=".dylib" Condition="$([MSBuild]::IsOSPlatform('OSX'))" />
+      <HostDylibSuffix Include=".so" Condition="!$([MSBuild]::IsOSPlatform('OSX'))" />
```

Comment:
this doesn't seem to be used, remove it?

Reply:
Ah I see, it was supposed to be used in the `--libclang=` further down.

Instead of this, please just duplicate the `_MonoAotCrossOffsetsToolParams` line and add the IsOSPlatform(OSX) condition directly there.